etc/validator.out

```diff
@@ -34,7 +33,6 @@ sites/udmi_site_model/out/devices/GAT-123/state_system.json
 sites/udmi_site_model/out/devices/GAT-123/validation_report.json
 sites/udmi_site_model/out/devices/SNS-4/config.json
 sites/udmi_site_model/out/devices/SNS-4/events_pointset.json
-sites/udmi_site_model/out/devices/SNS-4/events_system.json
```

Comment:
why are these going away?

Reply:
Temporarily disabling system events for virtual proxy devices until we identify useful metrics to send, as they are currently reporting JVM CPU and memory usage of the gateway, which is not beneficial and creates unnecessary network traffic.

Reply:
Okay, I just re-enabled it and removed only the metrics from the proxy devices. It seems that the sequencer's 1-second delay between event processing is causing a timeout in the 50-proxy-device setup.